src/audio/chain_dma.c

```diff
@@ -544,6 +542,8 @@ static int chain_task_init(struct comp_dev *dev, uint8_t host_dma_id, uint8_t li
 		break;
 	}
 
+	fifo_size = ALIGN_UP_INTERNAL(fifo_size, addr_align);
+
```

Comment:
the change looks good, but the fifo_size computation above looks quite questionable. There is NOTHING that says the host buffer size is 20ms. That might be an assumption of the OS that shall not be named, but for Linux the host buffer size can be modified at will with a modified hw_params.

This seems very very odd to me.

@jsarha @ujfalusi @kv2019i am I the only one thinking something's not quite right here?

Reply:
@plbossart For the L1 exit latency part, this should be independent of host size, so having the calculation here seems ok, but the second comment on position reporting seems indeed odd (without input on actual host buffer size). Of course on Linux side, we could limit buffer sizes if chain-dma is used, but that is definitely not done now (and might get come suprising app behaviour as otherwise SOF is quite relaxed w.r.t. buffer sizes towards the apps).